compiler/resolution/lowerIterators.cpp

```diff
@@ -1490,17 +1490,11 @@ expandBodyForIteratorInline(ForLoop*         forLoop,
                             bool&            addErrorArgToCall,
                             const SymbolMap& map);
 
-static void markLoopProperties(ForLoop* forLoop, BlockStmt* ibody,
-                               bool forVectorize) {
+static void markLoopProperties(ForLoop* forLoop, BlockStmt* ibody) {
   bool forIsOrderIndep = forLoop->isOrderIndependent();
   bool forHasHazard = forLoop->hasVectorizationHazard();
   auto llvmAttrs = forLoop->getAdditionalLLVMMetadata();
 
-  if (forVectorize) {
-    forLoop->orderIndependentSet(true);
```

Comment:
No, it gets used elsewhere when setting up iterators

Reply:
Interesting.  I wonder if that implies that this should be getting called here in more situations (but that's not worth worrying about until we find a bug that says otherwise)